test/test_tags.rb

```diff
@@ -5,12 +5,7 @@
 class TestTags < JekyllUnitTest
 
   def create_post(content, override = {}, converter_class = Jekyll::Converters::Markdown)
-    stub(Jekyll).configuration do
-      site_configuration({
-        "highlighter" => "rouge"
-      }.merge(override))
-    end
-    site = Site.new(Jekyll.configuration)
+    site = fixture_site({"highlighter" => "rouge"}.merge(override))
```

Comment:
I just kept the way it was before.
So you are saying we dont need the `{"highlighter" => "rouge"}` part?